package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "integration-nameOfYourIntegration",
-  "version": "1.0.2",
+  "name": "integration-UserLeap",
+  "version": "1.0.3",
```

Comment:
technically this is V1.0. of your integration.  it was just 1.0.2 of the example at the time you forked it